docs/i18n/en/docusaurus-plugin-content-docs/version-3.0.x.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
why is this 3.0.x.json?

Reply:
We didn't change this & we're not sure what version should it be. Kindly let us know, what's the correct version, so that we can modify this as well.

Reply:
why is this file created newly?
Similarly for other files of this type.

Reply:
We didn't create this file @avishnu.